package.json

```diff
@@ -1,22 +1,22 @@
 {
   "name": "get-port-cli",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Get an available port",
   "license": "MIT",
   "repository": "sindresorhus/get-port-cli",
   "author": {
     "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
-    "url": "sindresorhus.com"
+    "url": "https://sindresorhus.com/"
   },
   "bin": {
     "get-port": "cli.js"
   },
   "engines": {
-    "node": ">=4"
+    "node": ">=6"
```

Comment:
Don't do unrelated changes.

Reply:
This is not unrelated, the main package (`get-port`) requires this